app/code/Magento/InventoryInStorePickupAdminUi/view/adminhtml/templates/order/create/shipping/method/sources_form.phtml

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+use Magento\InventoryInStorePickupAdminUi\ViewModel\CreateOrder\SourcesForm;
+
+// @codingStandardsIgnoreFile
+/** @var SourcesForm $viewModel */
+$viewModel = $block->getData('view_model');
+if (!$viewModel) {
+    return '';
+}
+/** @var Magento\Backend\Block\Template $block */
+$sources = $viewModel->getPickupLocationsArray();
+$selectedSourceCode = $viewModel->getSelectedPickupLocationCode();
+?>
+<fieldset class="admin__fieldset payment-method"
+          id="shippinh_form_pickup_location_source"
+>
+    <div class="admin__field _required">
+        <label class="label admin__field-label" for="pickup_location_source">
+            <span><?= $block->escapeHtml(__('Pickup Location Source')) ?></span>
```

Comment:
Would be nice not to forget discussing naming for the Admin UI

Reply:
@novikor `Pickup Location Source` or `Pickup Source` or ... ?

Reply:
Yes